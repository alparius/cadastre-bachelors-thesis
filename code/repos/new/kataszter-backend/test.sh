#!/bin/bash

rm -f coverage.txt
echo 'mode: atomic' > coverage.txt

for d in $(go list ./... | grep -v testing); do
    go test -v -p 1 -race -covermode=atomic -coverprofile=coverage.out -coverpkg=$(go list ./...| grep -v testing | tr "\n" ",") $d
    if [ -f coverage.out ]; then
        tail -n +2 coverage.out >> coverage.txt || exit 255
        rm coverage.out
    fi
done
