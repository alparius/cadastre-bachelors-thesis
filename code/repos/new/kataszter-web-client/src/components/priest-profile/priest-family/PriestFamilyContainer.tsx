import React from "react";
import { Button, Card, Divider, Header, Icon } from "semantic-ui-react";

import { NewChild } from "../../../data/priest/Child";
import IPriestProfileData from "../../../data/priest/PriestProfileData";
import { NewSpouse } from "../../../data/priest/Spouse";
import BoldFM from "../../../util/styledMessages/BoldFM";
import ItalicFM from "../../../util/styledMessages/ItalicFM";
import ErrorMsg from "../../ErrorMsg";
import ProfilePlaceholder from "../../ProfilePlaceholder";
import ParentCard from "./ParentCard";
import PriestFamilyChildList from "./PriestFamilyChildList";
import PriestFamilySpouseList from "./PriestFamilySpouseList";

interface IPriestFamilyContainerProps {
    priest: IPriestProfileData;
    loading: boolean;
    showError: boolean;
    editMode: boolean;
    setPriest(newPriest: IPriestProfileData): void;
}

const PriestFamilyContainer: React.FC<IPriestFamilyContainerProps> = (props: IPriestFamilyContainerProps) => {
    const { priest, setPriest, loading, showError, editMode } = props;

    const handleNewChild = () => {
        if (priest) {
            if (priest.children) {
                priest.children.splice(0, 0, { ...NewChild });
                setPriest({
                    ...priest,
                    children: priest.children
                });
            } else {
                setPriest({
                    ...priest,
                    children: [{ ...NewChild }]
                });
            }
        }
    };

    const handleNewSpouse = () => {
        if (priest) {
            if (priest.spouses) {
                priest.spouses.splice(0, 0, { ...NewSpouse });
                setPriest({
                    ...priest,
                    spouses: priest.spouses
                });
            } else {
                setPriest({
                    ...priest,
                    spouses: [{ ...NewSpouse }]
                });
            }
        }
    };

    const handleChangeOnParent = (labelName: string, value: any) => {
        setPriest({ ...priest, [labelName]: value });
    };

    if (showError) {
        return <ErrorMsg errorMessage={"error.connection"} />;
    } else if (loading || !priest) {
        return <ProfilePlaceholder />;
    } else {
        return (
            <>
                <Header as="h2">
                    <ItalicFM id="profile.parents" />
                </Header>
                {priest.father.name || priest.mother.name || editMode ? (
                    <Card.Group stackable doubling itemsPerRow={2}>
                        <ParentCard label="father" parentData={priest.father} editMode={editMode} handleChangeOnParent={handleChangeOnParent} />
                        <ParentCard label="mother" parentData={priest.mother} editMode={editMode} handleChangeOnParent={handleChangeOnParent} />
                    </Card.Group>
                ) : (
                    <ItalicFM id="general.noData" />
                )}

                <Divider />

                <Header as="h2">
                    {priest.spouses && priest.spouses.length > 1 ? <ItalicFM id="profile.spouses" /> : <ItalicFM id="profile.spouse" />}
                </Header>
                {editMode && (
                    <Button id="addNewSpouse" icon basic color="violet" circular labelPosition="left" onClick={handleNewSpouse}>
                        <Icon name="add" color="violet" size="large" />
                        <BoldFM id="profile.addNewSpouse" />
                    </Button>
                )}

                <PriestFamilySpouseList priest={priest} editMode={editMode} setPriest={setPriest} />
                <Divider style={{ marginTop: "2em" }} />

                <Header as="h2">
                    {priest.children && priest.children.length > 1 ? <ItalicFM id="profile.children" /> : <ItalicFM id="profile.child" />}
                </Header>
                {editMode && (
                    <Button id="addNewChild" icon basic color="violet" circular labelPosition="left" onClick={handleNewChild}>
                        <Icon name="add" color="violet" size="large" />
                        <BoldFM id="profile.addNewChild" />
                    </Button>
                )}
                <PriestFamilyChildList priest={priest} editMode={editMode} setPriest={setPriest} />
            </>
        );
    }
};

export default PriestFamilyContainer;
