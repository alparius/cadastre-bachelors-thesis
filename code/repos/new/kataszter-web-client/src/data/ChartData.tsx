interface IChartData {
    key: any;
    data: any;
    counter: number;
}

export default IChartData;
