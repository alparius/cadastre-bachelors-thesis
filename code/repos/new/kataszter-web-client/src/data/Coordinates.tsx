interface ICoordinates {
    lat: number;
    lng: number;
}

export let NewCoordinates: ICoordinates = {
    lat: 0,
    lng: 0
};

export default ICoordinates;
