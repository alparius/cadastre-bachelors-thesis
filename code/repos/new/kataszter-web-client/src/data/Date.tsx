interface IDate {
    year: number;
    month: number;
    day: number;
    comment: string;
}

export let NewDate: IDate = {
    year: 0,
    month: 0,
    day: 0,
    comment: ""
};

export default IDate;
