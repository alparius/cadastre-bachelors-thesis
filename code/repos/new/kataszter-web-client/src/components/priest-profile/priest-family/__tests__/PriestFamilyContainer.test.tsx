/*===================
  PriestFamilyContainer.unit.test.tsx
 ===================*/
import React from "react";

import IPriestProfileData from "../../../../data/priest/PriestProfileData";
import { NewTestPriestProfileData } from "../../../../util/test/data/test-data";
import { createComponentWithIntlAndRouter, mountWithIntl } from "../../../../util/test/test-wrapper";
import PriestFamilyContainer from "../PriestFamilyContainer";

const handlers = {
    setPriest: jest.fn(),
    setEditMode: jest.fn()
};

const priest: IPriestProfileData = NewTestPriestProfileData;

describe("PriestFamilyContainer element", () => {
    afterEach(() => {
        handlers.setPriest.mockClear();
        handlers.setEditMode.mockClear();
    });

    it("should render a family container when the priest is not empty", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestFamilyContainer priest={priest} loading={false} showError={false} editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render a placeholder when the priest is empty", () => {
        const emptyPriestFamily: IPriestProfileData[] = [];
        const container = createComponentWithIntlAndRouter(
            <PriestFamilyContainer priest={emptyPriestFamily[0]} loading={false} showError={false} editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render an error message when showError is true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestFamilyContainer priest={priest} loading={false} showError editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render an error message when showError and loading are both true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestFamilyContainer priest={priest} loading showError editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should render a placeholder card when only loading is true", () => {
        const container = createComponentWithIntlAndRouter(
            <PriestFamilyContainer priest={priest} loading showError={false} editMode={false} {...handlers} />
        ).toJSON();
        expect(container).toMatchSnapshot();
    });

    it("should call setPriest if add new child Button pressed", () => {
        const container = mountWithIntl(<PriestFamilyContainer priest={priest} loading={false} showError={false} editMode {...handlers} />);
        container.find("#addNewChild").at(0).simulate("click");
        expect(handlers.setPriest).toHaveBeenCalledTimes(1);
    });

    it("should call setPriest if add new spouse Button pressed", () => {
        const container = mountWithIntl(<PriestFamilyContainer priest={priest} loading={false} showError={false} editMode {...handlers} />);
        container.find("#addNewSpouse").at(0).simulate("click");
        expect(handlers.setPriest).toHaveBeenCalledTimes(1);
    });
});
