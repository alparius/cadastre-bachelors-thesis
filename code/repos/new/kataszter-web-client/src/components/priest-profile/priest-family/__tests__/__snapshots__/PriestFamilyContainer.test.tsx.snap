// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriestFamilyContainer element should render a family container when the priest is not empty 1`] = `
Array [
  <h2
    className="ui header"
  >
    <i>
      Parents
    </i>
  </h2>,
  <div
    className="ui doubling stackable two cards"
  >
    <div
      className="ui fluid card"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <div
          className="header"
          id="header"
        >
          <h2
            className="ui header"
          >
            Pataki Miklos
          </h2>
        </div>
        <div
          className="description"
        >
          <div
            className="ui list"
            role="list"
          >
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Job
                    :
                  </b>
                   
                  Kutyaház tetőfedő
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of birth
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of death
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ui fluid card"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <div
          className="header"
          id="header"
        >
          <h2
            className="ui header"
          >
            Pataki Miklos
          </h2>
        </div>
        <div
          className="description"
        >
          <div
            className="ui list"
            role="list"
          >
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Job
                    :
                  </b>
                   
                  Kutyaház tetőfedő
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of birth
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of death
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ui divider"
  />,
  <h2
    className="ui header"
  >
    <i>
      Spouse
    </i>
  </h2>,
  <div
    className="ui doubling stackable two cards"
  >
    <div
      className="ui fluid card"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <div
          className="content"
        >
          <div
            className="header"
            id="header"
          >
            <h2
              className="ui header"
            >
              Házastárs
            </h2>
          </div>
        </div>
        <div
          className="description"
        >
          <div
            className="ui list"
            role="list"
          >
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of birth
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of marriage
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Job
                    :
                  </b>
                   
                  Szavazati joggal rendelkező
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Father
                    :
                  </b>
                   
                  Lajos
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Mother
                    :
                  </b>
                   
                  Laci
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="ui divider"
    style={
      Object {
        "marginTop": "2em",
      }
    }
  />,
  <h2
    className="ui header"
  >
    <i>
      Child
    </i>
  </h2>,
  <div
    className="ui doubling stackable two cards"
  >
    <div
      className="ui fluid card"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <div
          className="content"
        >
          <div
            className="header"
            id="header"
          >
            <h2
              className="ui header"
            >
              Gyerek
            </h2>
          </div>
        </div>
        <div
          className="description"
        >
          <div
            className="ui list"
            role="list"
          >
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Place of birth
                    :
                  </b>
                   
                  Krahorandia
                </div>
              </form>
            </div>
            <div
              className="item"
              onClick={[Function]}
              role="listitem"
            >
              <form
                className="ui form"
                onSubmit={[Function]}
              >
                <div
                  className="inline field"
                >
                  <b>
                    Date of birth
                    :
                  </b>
                   
                  <span
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                  >
                    2000.06.06
                  </span>
                </div>
              </form>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;

exports[`PriestFamilyContainer element should render a placeholder card when only loading is true 1`] = `
Array [
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
]
`;

exports[`PriestFamilyContainer element should render a placeholder when the priest is empty 1`] = `
Array [
  <div
    className="ui four column grid"
  >
    <div
      className="column"
    />
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="rectangular image"
        />
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="ui placeholder"
      >
        <div
          className="line"
        />
      </div>
    </div>
    <div
      className="column"
    />
  </div>,
  <div
    className="ui divider"
  />,
  <div
    className="ui four column grid"
  >
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      />
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      >
        <div
          className="ui placeholder"
        >
          <div
            className="line medium"
          />
        </div>
      </div>
      <div
        className="column"
      />
    </div>
  </div>,
]
`;

exports[`PriestFamilyContainer element should render an error message when showError and loading are both true 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;

exports[`PriestFamilyContainer element should render an error message when showError is true 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "7em",
    }
  }
>
  <h2
    className="ui header red"
  >
    Could not estabilish server connection.
  </h2>
  <h3
    className="ui header"
  >
    <a
      aria-current="page"
      className="hoverLink active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Back to the Home Page
    </a>
  </h3>
</div>
`;
